modules/oe_content_entity/tests/src/FunctionalJavascript/StakeholderUiTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Let's remove this test entirely (and the image asset too), we will test this in Behat once the project will be merged.

Reply:
done